doc/manuals/user/metadata.md

```diff
@@ -402,3 +403,34 @@ attributes using ID metadata:
     remove an specific attribute identified by ID (see DELETE attribute
     semantics [described in a previous
     section](append_and_delete.md#adding-and-removing-attributes-with-append-and-delete-in-updatecontext)).
+
+
+## Metadata in notifications
+
+By default, all custom (user) metadata are included in notifications. However, the field `metadata`
+can be used to filter the list. In addition, it can be used to specify that the following special
+metadata (not included by default) must be included.
+
+* previousValue
+* actionType
+
+Details about their meaning can be found in the "Special metadata in notifications" section in
+the NGSIv2 specification).
+
+Note that using the following
+
+```
+"metadata": [ "previousValue" ]
+```
+
+will cause to include `previousValue` but will remove any other metadata that
+attributes in the notification may have. If you want to get `previousValue`
+*and* any other "regular" metadata then use:
+
+```
+"metadata": [ "previousValue", "*" ]
+```
+
+Note that you can also use `"metadata": [ "*" ]` although it doesn't make much sense, as
+it gives the same result as not including `metadata` at all (remember that the default
+behaviour is to include all custom metadata).
```

Comment:
Fixed in 7a2e365 (although we haven't decided yet about "System metadata" I think "user" is better than "custom" in this point)